docs/workbench-playground-1-intro.md

```diff
@@ -0,0 +1,36 @@
+---
+id: workbench-playground-introduction
+title: Workbench
+sidebar_label: Workbench - Playground intro
+---
+
+# Playground walkthrough.
+
+The playground is an interactive environment where you can build and test serverless functions without having to create them with your namespace. You can access it by clicking on the `playground` button or type `playground` in the workbench.
+
+To display display the menu in the Workbench, run `help`.
+To remove the menu, run `clear`.
+
+## What you can do in the Playground?
+
+### Select the right programming language for you.
+
+At the top of the workbench you can click the button that says "JavaScript" and select from a wide range of programming languages. You can pick JavaScript, Typescript, Python, Swift, Java, Go, or PHP.
+
+![](https://github.com/JamieDawson/docs/blob/workbench-documents/static/img/playground_languages.png)
```

Comment:
As noted above, add `playground_languages.png` as a file to this PR and commit it, then reference it using a relative link.